### ISSUE TITLE
Encoding characters in turtle/ntriple

### ISSUE THREAD

**Issue by ncarboni** [innoq/iqvoc#362]
I have a problem with the encoding of the characters in turtle/n-triple. 
I am using english and french labels for the vocabulary' s terms. 
When one term have an accent (typically the french one), I can visualise it in the HTML,  in the full export as well as if I download the RDF.  
However, If I visualise the ttl or nt of the same term, I see it as such
For example, the term **Déformation**

in the RDF file downloaded from the website appear as
```
<skos:prefLabel xml:lang="en">Deformation</skos:prefLabel>
 <skos:prefLabel xml:lang="fr">Déformation</skos:prefLabel>
```
while in Turtle

```
skos:prefLabel "Deformation"@en;
skos:prefLabel "DÃ©formation"@fr;
```
 and in N-triple
```
<http://website/vocab/_bb1ba8aa> <http://www.w3.org/2004/02/skos/core#prefLabel> "Deformation"@en .
<http://website/vocab/_bb1ba8aa> <http://www.w3.org/2004/02/skos/core#prefLabel> "DÃ©formation"@fr .
```

I sincerely do not understand why. At the beginning I though it was a problem of locale, but everything should be in place (below the result of `locale -a`):

> C
> C.UTF-8
> de_DE.utf8
> en_AG
> en_AG.utf8
> en_AU.utf8
> en_BW.utf8
> en_CA.utf8
> en_DK.utf8
> en_GB.utf8
> en_HK.utf8
> en_IE.utf8
> en_IN
> en_IN.utf8
> en_NG
> en_NG.utf8
> en_NZ.utf8
> en_PH.utf8
> en_SG.utf8
> en_US.utf8
> en_ZA.utf8
> en_ZM
> en_ZM.utf8
> en_ZW.utf8
> fr_FR
> fr_FR.iso88591
> fr_FR.utf8
> POSIX

Any ideas?

**Comment by philippneugebauer** [comment]
will be fixed through https://github.com/innoq/iq_rdf/pull/13

**Comment by mjansing** [comment]
Fixed by bumping iq_rdf to 0.1.16 on master. Fix will be included in v4.11.2. 

Thank you.
